readthedocs/templates/404.html

```diff
@@ -1,84 +1,2 @@
-
-  <p>
-    {% blocktrans with project_slug=project.slug request_path=request.path %}You are browsing the documentation of "{{ project_slug }}". The page you are looking for at <code>{{ request.path }}</code> was not found.{% endblocktrans %}
-  </p>
-
-  <p>
-    {% blocktrans %}Documentation changes over time. Pages are moved around. You can try to navigate to <a href="/">the index page</a> of the project and use its navigation or search function to find a similar page.{% endblocktrans %}</p>
-
-  <h3>{% trans "Are you the project owner?" %}</h3>
-
-  <p>{% trans "Here are some tips to address 404 errors:" %}</p>
-
-  <ul>
-    <li>{% trans "Use custom 404 pages:" %} <a href="https://docs.readthedocs.io/en/latest/hosting.html#custom-not-found-404-pages">{% trans "Read more" %}»</a></li>
-    <li>{% trans "Create redirects when you move contents:" %} <a href="https://docs.readthedocs.io/en/stable/user-defined-redirects.html">{% trans "Read more" %}»</a></li>
-    <li>{% trans "Use rediraffe to manage redirects in Sphinx:" %} <a href="https://pypi.org/project/sphinxext-rediraffe/">{% trans "Read more" %}»</a></li>
-  </ul>
-
-{% elif "_proxito_404_" in request.path %}
-
-  {# we are looking at a 404 generated by proxito, we then know that the project itself doesn't exist or hasn't been built #}
-
-  <p>
-    {% blocktrans with project_slug=project_slug %}The project "{{ project_slug }}" that you are looking for either does not exist, has been deleted or does not yet have any valid builds.{% endblocktrans %}
-  </p>
-
-{% else %}
-
-  <p>{% trans "You have encountered a 404 on Read the Docs. You are either looking for a page that does not exist or a project that has been removed." %}</p>
-
-  <p>{% trans "If you believe this is a technical error, please contact our support:" %} <a href="{% url 'support' %}">{% trans "Go to support" %}»</a></p>
-
-{% endif %}
-
-<pre style="line-height: 1.25; white-space: pre;">
-
-        \          SORRY            /
-         \                         /
-          \    This page does     /
-           ]   not exist yet.    [    ,'|
-           ]                     [   /  |
-           ]___               ___[ ,'   |
-           ]  ]\             /[  [ |:   |
-           ]  ] \           / [  [ |:   |
-           ]  ]  ]         [  [  [ |:   |
-           ]  ]  ]__     __[  [  [ |:   |
-           ]  ]  ] ]\ _ /[ [  [  [ |:   |
-           ]  ]  ] ] (#) [ [  [  [ :===='
-           ]  ]  ]_].nHn.[_[  [  [
-           ]  ]  ]  HHHHH. [  [  [
-           ]  ] /   `HH("N  \ [  [
-           ]__]/     HHH  "  \[__[
-           ]         NNN         [
-           ]         N/"         [
-           ]         N H         [
-          /          N            \
-         /           q,            \
-        /                           \
-</pre>
-{% endblock %}
+{% extends "errors/404/base.html" %}
+{# this is a deprecated template, will be removed. Please use one of the targeted 404s in templates/404/ #}
```

Comment:
Why we don't just remove it?

Reply:
I'll have a look and see. But I prefer removing it in a later change set, simply to reduce the potential impact of this change.

Reply:
Two gotchas of why I don't feel fully comfortable on removing this:

* `404.html` is the default 404 template for Django's default 404 view.
* I'm not sure about other projects that might somehow anticipate this file to exist, for instance I haven't looked in readthedocs-corporate.

It seems rather harmless to leave it like this.